rotkehlchen/accounting/structures/processed_event.py

```diff
@@ -191,7 +191,7 @@ def deserialize_from_db(cls: Type[T], timestamp: Timestamp, stringified_json: st
                 notes=data['notes'],
                 location=Location.deserialize(data['location']),
                 timestamp=timestamp,
-                asset=Asset(data['asset']),
+                asset=Asset(data['asset']).check_existence(),
```

Comment:
Hmmm this is from the `data` part of the transient DB's pnl_events table. I think it's good to have a check here as his is a json dict saved in the transient DB and the DB's coherency may break there if somethign weird happens. So here can keep it.

But you are introducing a potential unhandled error here. You will need to handlethe potential `UnknownAsset` exception in the except below and raise a `DeserializationError` like it does for `KeyError`